internal/distro/rhel8/pipelines.go

```diff
@@ -698,7 +698,15 @@ func osPipeline(t *imageType,
 			},
 		}))
 	}
-
+	var greenbootStageOptions *osbuild.GreenbootConfig
+	greenbootConfig := &blueprint.GreenbootCustomization{
```

Comment:
this is very confusing again @say-paul, why initializing something to something which we later convert anyway? at the end of the day you only need a `*osbuild.GreenbootConfig` and `GreenbootStageOptions` only adds to the complexity of the code. Moreover, `MonitorServices` is just a normal `[]string` so why the need for the extra function to convert/assign, take a look here https://github.com/osbuild/osbuild-composer/pull/2906/files#r952675661

Reply:
seems i was too influenced with the stage_options function , now i get it, thanks for simplifying things.